back-end/dao/intentdao.spec.js

```diff
@@ -82,7 +82,8 @@ describe("IntentDao", () => {
         Intent.find.mockImplementation(() => [1]);
         Intent.mockImplementationOnce(() => {save: jest.fn()});
         await dao.postIntents(fakeIntents, true);
-        expect(Intent.find).not.toHaveBeenCalled();
+
```

Comment:
+1

Reply:
Actually I wanted to ask do we not need this line anymore?